realm/realm-library/src/main/java/io/realm/internal/OsObject.java

```diff
@@ -156,4 +243,25 @@ private void notifyChangeListeners(String[] changedFields) {
     private native void nativeStartListening(long nativePtr);
 
     private native void nativeStopListening(long nativePtr);
+
+    private static native long nativeCreateNewObject(long sharedRealmPtr, long tablePtr);
+
+    private static native long nativeCreateRow(long sharedRealmPtr, long tablePtr);
+
+
+    private static native long nativeCreateNewObjectWithPrimaryKey(long sharedRealmPtr,
+                                                                   long tablePtr, long pk_column_index,
+                                                                   long primaryKeyValue, boolean isNullValue);
+
+    private static native long nativeCreateRowWithPrimaryKey(long sharedRealmPtr,
```

Comment:
The meanings of the return values of `nativeCreateNewObject*` and `nativeCreateRow*` is a bit confusing.
`nativeCreateNewObject*` returns a pointer to native `Row` object and `nativeCreateRow*` returns an index of the created row in the `Table` (this method does not create `Row` instance).

Shouldn't we add a javadoc comment at least?